e2e/transactions/transactions_test.go

```diff
@@ -591,6 +593,9 @@ func (s *TransactionsTestSuite) TestCompleteMultipleQueuedTransactions() {
 			)
 		}
 	}
+
+	wg := sync.WaitGroup{}
```

Comment:
`var wg sync.WaitGroup` is more idiomatic in this case.

Reply:
Why?

Reply:
It's just more natural in Go world. Check out documentation: https://golang.org/pkg/sync/#WaitGroup and an example.

Reply:
I can change this, hovewer code style is out of scope in CR.

Reply:
Would prefer `var wg sync.WaitGroup` like already used in `transactions_test.go` too, but it's no show-stopper.

Reply:
It was... https://github.com/status-im/status-go/pull/456/commits/481e7c968a86ddd91f9f29aa1b7dd623df3a5027
I glad to remake its declaration in any way :+1: but it looks like we all want different things )
@adambabik